chore(autoware.repos): add managed transform buffer

amadeuszsz opened the pull request:
## Description

See https://github.com/autowarefoundation/autoware/issues/5385

**Order of PRs merge:**
- https://github.com/autowarefoundation/autoware/pull/5611 `<---`
- https://github.com/autowarefoundation/autoware_utils/pull/41
- https://github.com/autowarefoundation/autoware_universe/pull/9197
- https://github.com/autowarefoundation/autoware_core/pull/360
- https://github.com/autowarefoundation/autoware_launch/pull/1393

## How was this PR tested?

## Notes for reviewers

None.

## Effects on system behavior

None.

amadeuszsz committed:
chore(autoware.repos): add managed transform buffer

Signed-off-by: Amadeusz Szymko <amadeusz.szymko.2@tier4.jp>

amadeuszsz committed:
Merge branch 'main' into chore/add-managed_transform_buffer

youtalk reviewed (commented):
Please fix the `version`.

amadeuszsz committed:
chore(autoware*.repos): fix managed transform buffer version

Signed-off-by: Amadeusz Szymko <amadeusz.szymko.2@tier4.jp>

amadeuszsz commented:
> Please fix the `version`.

@youtalk 
Addressed in https://github.com/autowarefoundation/autoware_universe/pull/9197/commits/d431b4edb07d6c318ba16b9ee25b2e61cad9e1e2 and https://github.com/autowarefoundation/autoware/pull/5611/commits/75ab54386caa292bcc972ab2a40dc4f5688e7a23. I wasn't sure if repository top name has to align with ROS build farm naming convention, but also addressed.

youtalk reviewed (approved):
The naming and versioning look much better. LGTM
